src/LightGraphs.jl

```diff
@@ -272,6 +272,7 @@ include("vertexcover/degree_vertex_cover.jl")
 include("vertexcover/random_vertex_cover.jl")
 include("Experimental/Experimental.jl")
 include("Parallel/Parallel.jl")
+include("simplegraphs/generators/staticgraphs.jl")
```

Comment:
You shouldn't be changing includes in this PR. Something else is wrong if you need to do this...

Reply:
I tried to do an experiment, i can’t know why it yields an error.